sauna/plugins/ext/network.py

```diff
@@ -0,0 +1,130 @@
+import time
+
+from sauna.plugins.base import PsutilPlugin
+from sauna.plugins import PluginRegister
+
+my_plugin = PluginRegister('Network')
+
+
+@my_plugin.plugin()
+class Network(PsutilPlugin):
+
+    def __init__(self, config):
+        super().__init__(config)
+
+    @my_plugin.check()
+    def upload_kbytes_speed(self, check_config):
+        ul, _, _, _ = self.get_network_data(
+            interface=check_config['interface'])
+        ul = round(ul, 2)
+        return (
+            self._value_to_status_less(ul, check_config),
+            'Upload speed: {} KB/s'.format(ul)
+        )
+
+    @my_plugin.check()
+    def download_kbytes_speed(self, check_config):
+        _, dl, _, _ = self.get_network_data(
+            interface=check_config['interface'])
+        dl = round(dl, 2)
+        return (
+            self._value_to_status_less(dl, check_config),
+            'Download speed: {} KB/s'.format(dl)
+        )
+
+    @my_plugin.check()
+    def cumulative_kbytes_speed(self, check_config):
```

Comment:
I think i'll delete all "cumulative" checks, I saw this in other tool but i'm not sure it's usefull